applications/nrf_desktop/README.rst

```diff
@@ -708,25 +827,37 @@ Precision Aim button
     * |nRF_Desktop_cancel_operation|
 
 Desktop mouse
-~~~~~~~~~~~~~
++++++++++++++
 
 The following predefined buttons are assigned to peer control operations for the desktop mouse:
 
 Scroll wheel button
     * Long-press to initialize and confirm the peer erase.
       The scroll must be pressed before the mouse is powered up.
 
-        * |nRF_Desktop_confirmation_effect|
+      .. figure:: /images/nrf_desktop_desktop_mouse_side.svg
```

Comment:
**Image edit:**
Please remove the debug caption from this image.

Also, @pdunaj, about middle button vs scroll wheel button. I think "scroll wheel button" is less confusing if the mouse has a button under the scroll. Some mice have DPI buttons located under the scroll wheel or above it, and they could also be considered "middle buttons".